Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsentController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using IdentityServer4.Services;
+using IdentityServer4.Stores;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Promact.Oauth.Server.Models.IdentityServer4;
+using Promact.Oauth.Server.Services;
+using System.Threading.Tasks;
+
+namespace Promact.Oauth.Server.Controllers
+{
+    /// <summary>
+    /// This controller processes the consent UI
+    /// </summary>
+    [ServiceFilter(typeof(SecurityHeadersAttribute))]
+    public class ConsentController : BaseController
+    {
+        private readonly Services.IConsentService _consent;
+
+        public ConsentController(IIdentityServerInteractionService interaction, IClientStore clientStore,
+            IResourceStore resourceStore, ILogger<ConsentController> logger, Services.IConsentService consent)
+        {
+            _consent = consent;
+            //_consent = new IConsentService(interaction, clientStore, resourceStore, logger);
+        }
+
+        /// <summary>
+        /// Shows the consent screen
+        /// </summary>
+        /// <param name="returnUrl">redirect url</param>
+        /// <returns>view of allowed scope</returns>
+        [Microsoft.AspNetCore.Mvc.HttpGet]
```

Comment:
remove entire namespace used only HttpGet instead of Microsoft.AspNetCore.Mvc.HttpGet